app/assets/javascripts/leaflet.map.js

```diff
@@ -9,10 +14,58 @@ L.extend(L.LatLngBounds.prototype, {
   }
 });
 
+L.MapTiler = L.MaplibreGL.extend({
+  options: {
+    maxZoom: 23,
+    attribution: "© <a href=\"https://www.openstreetmap.org/copyright\" target=\"_blank\">OpenStreetMap</a> contributors. Tiles courtesy of <a href=\"http://memomaps.de/\" target=\"_blank\">MeMoMaps</a>"
+  },
+  onAdd: function (map) {
+    function update3dMapUrl() {
+      var el = document.getElementById("terrain_3d_url");
+
+      if (el) {
+        var c = map.getCenter();
+        var zoom = map.getZoom() - 1;
+        var precision = Math.ceil(((zoom * Math.LN2) + Math.log(256 / 360)) / Math.LN10);
+
+        el.href = el.dataset.hrefTemplate
+          .replace("{lang}", I18n.locale.replace(/-.*/, ""))
+          .replace("{pos}", zoom + "/" + c.lat.toFixed(precision) + "/" + c.lng.toFixed(precision) + "/30.00/60.00");
+      }
+    }
+
+    map._update3dMapUrl = update3dMapUrl;
+    setTimeout(update3dMapUrl);
+    map.on("moveend", update3dMapUrl);
```

Comment:
You don't need any of this added in this commit if you remove it in the next one.